cmd/logout.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2016 Dropbox, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+)
+
+const revokeURL = "https://api.dropboxapi.com/2/auth/token/revoke"
+
+// Sends a POST request to /auth/token/revoke to revoke the given API token.
+func revoke(tok string) error {
```

Comment:
@diwakergupta Where is this `TokenRevoke` method defined? I can't find anything like that searching the [source](https://github.com/dropbox/dropbox-sdk-go-unofficial/search?utf8=✓&q=TokenRevoke) or [godoc](http://godoc.org/github.com/dropbox/dropbox-sdk-go-unofficial). The [auth package](https://github.com/dropbox/dropbox-sdk-go-unofficial/blob/master/auth/interface.go) is empty, and the [`Api`](https://github.com/dropbox/dropbox-sdk-go-unofficial/blob/master/impl.go#L39) interface that [`Client`](https://github.com/dropbox/dropbox-sdk-go-unofficial/blob/master/util.go#L69) returns doesn't mention it either.

I used raw HTTP requests the first time around since I couldn't find a method for the revoke API, but I'll gladly use an official method if it exits. If it's _supposed_ to exist but hasn't been written yet I can even make a PR for the SDK to add it.